playwright.config.ts

```diff
@@ -0,0 +1,78 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './__playwright-tests__',
+  use: { baseURL: 'http://localhost:3000' },
+  /* Run tests in files in parallel */
```

Comment:
Can we delete all the commented out config?

Reply:
Yup, I'll remove it.